simple_qa_rnn/relation_prediction/hyperparameter_tuning.py

```diff
@@ -10,8 +10,8 @@
     dropout = uniform(0.5, 0.6)
     clip = uniform(0.6, 0.7)
 
-    command = "python train.py --cuda --device 1 --dev_every 500 --log_every 250 --batch_size 128 " \
-                "--epochs {} --lr {} --d_hidden {} --n_layers {} --dropout_prob {} --clip {} >> " \
+    command = "python train.py --cuda --gpu 1 --dev_every 500 --log_every 250 --batch_size 32 " \
```

Comment:
I don't mind if you just change it to 0. Some torchtext code does this. You can also make it a parameter of this script.